<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StyleEditor</class>
 <widget class="QWidget" name="StyleEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupStyles">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Styles</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QListWidget" name="stylesNames"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupFont">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Apparence</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Font</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QFontComboBox" name="fontName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Size</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="fontSize">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Color</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btnColor">
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>stylesNames</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>StyleEditor</receiver>
   <slot>styleSelected(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fontName</sender>
   <signal>currentFontChanged(QFont)</signal>
   <receiver>StyleEditor</receiver>
   <slot>apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fontSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>StyleEditor</receiver>
   <slot>apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnColor</sender>
   <signal>released()</signal>
   <receiver>StyleEditor</receiver>
   <slot>chooseColour()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>styleSelected(int)</slot>
  <slot>apply()</slot>
  <slot>chooseColour()</slot>
 </slots>
</ui>
